cpp/benchmarks/string/convert_floats_benchmark.cpp

```diff
@@ -0,0 +1,131 @@
+  auto& numbers = number_arrays[array_size];
+  if (numbers.size() == 0) {
+    numbers.reserve(array_size);
+    cudf::test::UniformRandomGenerator<FloatType> rand_gen(std::numeric_limits<FloatType>::min(),
+                                                           std::numeric_limits<FloatType>::max());
+    std::generate_n(
+      std::back_inserter(numbers), array_size, [&rand_gen]() { return rand_gen.generate(); });
+  }
+  return numbers;
+}
+
+template <class FloatType>
+static std::vector<std::string> get_floats_numbers_as_string(int64_t array_size)
+{
+  std::vector<std::string> numbers_str(array_size);
+  const auto& numbers = get_float_numbers<FloatType>(array_size);
+  std::transform(
+    numbers.begin(), numbers.end(), numbers_str.begin(), [](auto x) { return std::to_string(x); });
+  return numbers_str;
+}
+
+}  // anonymous namespace
+
+class StringToFloatNumber : public cudf::benchmark {
+};
+template <cudf::type_id float_type>
+void convert_to_float_number(benchmark::State& state)
+{
+  const auto& h_strings   = get_floats_numbers_as_string<double>(state.range(0));
+  const auto strings_size = std::accumulate(
+    h_strings.begin(), h_strings.end(), std::size_t{0}, [](std::size_t size, const auto& str) {
+      return size + str.length();
+    });
+
+  cudf::test::strings_column_wrapper strings(h_strings.begin(), h_strings.end());
+  const auto strings_view = cudf::strings_column_view(strings);
+
+  for (auto _ : state) {
+    cuda_event_timer raii(state, true);  // flush_l2_cache = true, stream = 0
+    volatile auto results = cudf::strings::to_floats(strings_view, cudf::data_type{float_type});
+  }
+
+  state.SetBytesProcessed(state.iterations() * strings_size);
+}
+
+class StringFromFloatNumber : public cudf::benchmark {
+};
+template <class FloatType>
+void convert_from_float_number(benchmark::State& state)
+{
+  const auto& h_floats   = get_float_numbers<FloatType>(state.range(0));
+  const auto floats_size = h_floats.size() * sizeof(FloatType);
+
+  cudf::test::fixed_width_column_wrapper<FloatType> floats(h_floats.begin(), h_floats.end());
+  const auto floats_view = cudf::column_view(floats);
+
+  for (auto _ : state) {
+    cuda_event_timer raii(state, true);  // flush_l2_cache = true, stream = 0
+    volatile auto results = cudf::strings::from_floats(floats_view);
+  }
+
+  state.SetBytesProcessed(state.iterations() * floats_size);
+}
+
+#define CV_TO_FLOATS_BENCHMARK_DEFINE(name, float_type_id)                  \
+  BENCHMARK_DEFINE_F(StringToFloatNumber, name)(::benchmark::State & state) \
+  {                                                                         \
+    convert_to_float_number<float_type_id>(state);                          \
+  }                                                                         \
+  BENCHMARK_REGISTER_F(StringToFloatNumber, name)                           \
+    ->RangeMultiplier(1 << 2)                                               \
```

Comment:
just a suggestion, '4' seems clearer to me
```suggestion
    ->RangeMultiplier(4)                                                    \
```